### PR TITLE
Fix "Hide points in add_point_labels is not working"

### LINKED ISSUE #1077: Hide points in add_point_labels is not working
Hi, Team! `add_point_label` is showing points even when `show_points=False`. I fixed this bug. See https://github.com/pyvista/pyvista/pull/1068/commits/64b4cea2e8aa3283ade34fab7b3cd303a0e2c459, please.

### PR DESCRIPTION
**Pull request by rodrigomologni**
### Overview

Resolves #1077 

### Details

- Fix to not show points if `show_points=False` in `add_point_labels`
- Update `test_point_labels` to verify the fix

Before:

image

After:

image

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -3273,15 +3273,11 @@ def add_point_labels(self, points, labels, italic=False, bold=True,
...
-        if show_points:
-            style = 'points'
-        else:
-            style = 'surface'
-        self.add_mesh(vtkpoints, style=style, color=point_color,
-                      point_size=point_size, name=f'{name}-points',
-                      pickable=pickable,
-                      render_points_as_spheres=render_points_as_spheres,
-                      reset_camera=reset_camera)
+        if show_points is True:
```
Comment (akaszynski) [inline pyvista/plotting/plotting.py:3276]: Remove redundant `is True`:
```suggestion
        if show_points:
```

**Comment by akaszynski** [comment]
Good fix.  Not sure why we implemented that with `style='surface'`.

One small note: try to use the branch naming conventions in [Contributing](https://docs.pyvista.org/extras/developer_notes.html).  It makes it easier to track which branches I can delete (which doesn't matter because it's a remote branch anyway).

Thanks again for the fix!

**Review by akaszynski — APPROVED** [review]

**Comment by rodrigomologni** [comment]
> Good fix. Not sure why we implemented that with style='surface'.

Thank you! I hope nobody has fixed the wrong method! :sweat_smile:

> One small note: try to use the branch naming conventions in Contributing. It makes it easier to track which branches I can delete (which doesn't matter because it's a remote branch anyway).

Sorry! I'm learning to use Git. First, I created the branch `fix` to update `add_point_labels`. Next, I created the branch `testing` to update `test_point_labels`. I thought that both branches would be sent. I'll get it right in the next pull request! :wink:

**Comment by akaszynski** [comment]
Regardless, nicely done.  Starting to contribute to open source using git isn't easy to start, so I appreciate your work.  Thanks again!

**Comment by rodrigomologni** [comment]
Thanks! I intend to contribute more. I thank you for creating PyVista. Great job. I'm using a lot! :clap:

**Pull request merged**